core-client/Cargo.toml

```diff
@@ -8,7 +8,7 @@ keywords = ["jsonrpc", "json-rpc", "json", "rpc", "serde"]
 license = "MIT"
 name = "jsonrpc-core-client"
 repository = "https://github.com/paritytech/jsonrpc"
-version = "12.0.0"
+version = "13.0.0"
```

Comment:
Do we really need `13.0`? Can't we just bump to `12.1`? Are there any breaking changes?

Reply:
If there is a bug in handling existing #[pubsub] macros it will break builds.

Reply:
Hmm, but is there?

Reply:
@dvc94ch true but that would be an unintentional "breaking" change, the API itself should still be compatible.

@tomusdrw so `core-client` and `derive` should both go to `12.1` then.?

Reply:
I'd rather keep all creates version aligned. So if we feel there is a reason to bump to 13.0 let's do that, but if there are no breaking changes that we know of, let's stick to 12.1 imho.

Reply:
Actually there is also an addition to `core` https://github.com/paritytech/jsonrpc/pull/436/files#diff-77e9ef3aa593be6048a014ed42ef10dbR34

So upgrade all crates to 12.1?